packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1386,25 +1433,65 @@ export const components: ThemeOptions["components"] = {
   MuiRadio: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <></>,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        "&:hover": {
+        width: theme.spacing(4),
```

Comment:
Same border variable and width/height value replacement suggestions here as with Checkbox.